src/test/java/org/sagebionetworks/bridge/sdk/integration/IntentToParticipateTest.java

```
@@ -120,11 +121,66 @@ public void testIntentToParticipate() throws Exception {
             
             ConsentStatus status = session.getConsentStatuses().get(IntegTestUtils.STUDY_ID);
             assertTrue(status.isConsented());
+        } catch(Exception e) {
+            fail("This should not have thrown an exception");
```

Don't do this. This masks any exception thrown by the test as well as any assert.

---

Removed